apps/website/gatsby-browser.ts

```diff
@@ -1,3 +1,5 @@
+import './node_modules/@custom/ui/build/styles.css';
```

Comment:
I saw that previously we imported it from both gatsby-browser and gatsby-ssr. Now it's gatsby-browser only. Not a mistake?

Reply:
It was a mistake before.